packaging/opensuse/snapd.spec

```diff
@@ -23,7 +23,7 @@
 # Enable AppArmor on openSUSE Tumbleweed (post 15.0) or higher
 # N.B.: Prior to openSUSE Tumbleweed in May 2018, the AppArmor userspace in SUSE
 # did not support what we needed to be able to turn on basic integration.
-%if 0%{?suse_version} >= 1550
+%if 0%{?suse_version} >= 1500
```

Comment:
IIRC this had something to do with use of `change_profile unsafe`, but I could be mistaken. Please make sure that Leap has a new enough apparmor for this (the version compare with Ubuntu may not be good enough since Ubuntu sometimes SRUd patches).

Reply:
We have explicit check for this in snapd https://github.com/snapcore/snapd/blob/master/sandbox/apparmor/apparmor.go#L219-L229 where we try to parse a profile containing `change_profile unsafe /**,`. Provided the check is sufficient, snapd on Leap reports:
```
AppArmor status: apparmor is enabled but some kernel features are missing: dbus
```
The spread suite appears green too, so hopefully things are sufficiently recent and usable. OTOH the spread test suite can only explore so much of the functionality. I'm willing to push this to our openSUSE system:snappy repo for the sake getting some wider testing with users.